client/client_test.go

```diff
@@ -168,15 +159,16 @@ func (s *TestSuite) TestChannels(t *C) {
 	// Start send/recv chans, but idle until successful Connect.
 	ws.Start()
 	defer ws.Stop()
+	defer ws.Disconnect()
 
 	ws.Connect()
-	c := <-mock.ClientConnectChan
+	c := <-s.server.ClientConnectChan
 	<-ws.ConnectChan()
 
 	// API sends Cmd to client.
 	cmd := &proto.Cmd{
 		User: "daniel",
-		Ts:   time.Now(),
+		Ts:   time.Now().UTC(),
```

Comment:
We assume everywhere that time should be in UTC, so some tests failed when time was used without UTC timezone.